### PR TITLE
Clear button for search

### LINKED ISSUE #1265: Clear button on search bar
https://trello.com/c/63Jf8Ddk/15-user-is-able-to-clear-their-search-input

https://zpl.io/2pOWe8Q
- **Given** user is using the search feature
- **When** they begin typing
- **Then** they see an X appear beside their input
- **And when** when they click on that X
- **Then** the search input is cleared 
- **And** the search results are cleared too

### PR DESCRIPTION
**Pull request by sastels**
resolves #1265

### CODE REVIEW

**Comment by sastels** [comment]
[User is able to clear their search input](https://trello.com/c/63Jf8Ddk/15-user-is-able-to-clear-their-search-input)

**Comment by sastels** [comment]
[User is able to clear their search input](https://trello.com/c/63Jf8Ddk/15-user-is-able-to-clear-their-search-input)

File: components/search_box.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // from: https://raw.githubusercontent.com/UKHomeOffice/govuk-react/master/components/search-box/src/index.js
-import React from "react";
+import React, { Component } from "react";
 import PropTypes from "prop-types";
 import styled from "react-emotion";
 import SearchIcon from "@material-ui/icons/Search";
+import ClearIcon from "@material-ui/icons/Clear";
```
Comment (smcmurtry) [inline components/search_box.js]: The icon in the designs is different. Were you thinking that would be a later PR?

Comment (smcmurtry) [reply components/search_box.js]: The correct icon is "cancel" in the Navigation section here: https://material.io/tools/icons/?icon=highlight_off&style=round

**Review by smcmurtry — APPROVED** [review]
Nice work!!

**Pull request merged**